kf/bpf.go

```diff
@@ -494,8 +494,8 @@ func (b *BPF) UpdateArgs(ifaceName, direction string) error {
 	UpdateCmd := execCommand(cmd, args...)
 	if err := UpdateCmd.Start(); err != nil {
 		stats.Incr(stats.BPFUpdateFailedCount, b.Program.Name, direction, ifaceName)
-		log.Info().Err(err).Msgf("user mode BPF program failed - %s", b.Program.Name)
-		return fmt.Errorf("failed to start : %s %v", cmd, args)
+		log.Warn().Err(err).Msgf("user mode BPF program failed - %s", b.Program.Name)
+		return fmt.Errorf("failed to start : %s %v %v", cmd, args, err)
```

Comment:
`fmt.Errorf()` returns an error, should we create that error first, log that error and then return it? While adding the message text to the error.
```suggestion
		err = fmt.Errorf("failed to start user mode BPF program failed -%s: %s %v %v", b.Program.Name, cmd, args, err)
		log.Warn().Err(err)
		return err
```

Reply:
Done